test/api/core/pluginContext/pluginContext.test.js

```diff
@@ -44,15 +44,15 @@ describe('Plugin Context', () => {
       const Koncorde = require('koncorde');
 
       should(context.constructors).be.an.Object().and.not.be.empty();
-      should(context.constructors.Dsl).be.a.Function();
+      should(context.constructors.Koncorde).be.a.Function();
       should(context.constructors.Koncorde).be.a.Function();
```

Comment:
Why this line is repeated twice ?

Reply:
uh... because I was a bit too hasty when I fixed the deprecation warnings and I opted for the wrong solution.

Gimme a few minutes.

Reply:
@Yoann-Abbes > Fixed. Instead of ignoring the deprecated property (which led to `Koncorde` being tested twice -_-), I properly mocked the deprecation tool, preventing it from printing unwanted messages during unit tests.

And since I mocked it, I added a unit test verifying that we properly deprecate the `Dsl` property in the plugins context.